torchtune/models/clip/_component_builders.py

```diff
@@ -0,0 +1,74 @@
+from typing import List, Optional, Callable
+
+import torch
+from torchtune.modules import VisionTransformer, CLSProjection
+from torchtune.models.clip._position_embeddings import TokenPositionalEmbedding, TiledTokenPositionalEmbedding, TilePositionalEmbedding
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+def clip_vision_encoder(
```

Comment:
I think this should also be a public API, no? (And if so, please add a docstring)